src/release.rs

```diff
@@ -0,0 +1,148 @@
+use {
+    crate::boxed_error,
+    git2::Repository,
+    log::*,
+    std::{error::Error, fmt::Display, path::PathBuf, str::FromStr, time::Instant},
+};
+
+#[derive(Debug)]
+pub enum DeployMethod {
+    Local,
+    Tar,
+    Skip,
+}
+
+impl Display for DeployMethod {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            DeployMethod::Local => write!(f, "local"),
+            DeployMethod::Tar => write!(f, "tar"),
+            DeployMethod::Skip => write!(f, "skip"),
+        }
+    }
+}
+
+impl FromStr for DeployMethod {
+    type Err = ();
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "local" => Ok(DeployMethod::Local),
+            "tar" => Ok(DeployMethod::Tar),
+            "skip" => Ok(DeployMethod::Skip),
+            _ => Err(()),
+        }
+    }
+}
```

Comment:
Feel free to use strum macros to avoid implementing these by hand: https://docs.rs/strum/latest/strum/

I think you'll just need `EnumString` and `IntoStaticString`. You can see a few instances of it in the token CLI https://github.com/solana-labs/solana-program-library/blob/ce8e4d565edcbd26e75d00d0e34e9d5f9786a646/token/cli/src/clap_app.rs#L81